### PR TITLE
Implement deleting user account by username

### PR DESCRIPTION
**Pull request by anastasiia-bilova**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -392,7 +413,7 @@ $ curl http://localhost:8000/block-producers/collection/ -H "Content-Type: appli
 ```bash
 $ curl -X PUT http://localhost:8000/block-producers/ \
      -H "Content-Type: application/json" \
-     -H "Authorization: JWT eyJ0e....eyJ1c2VyX2....sOx4S9zpC..." \
+     -H "Authorization: JWT eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJ1c2VyX2lkIjo2LCJ1c2VybmFtZSI6ImRteXRyby5zdHJpbGV0c2t5aSIsImV4cCI6MTU2NDQ5NjI4OSwiZW1haWwiOiJkbXl0cm8uc3RyaWxldHNreWlAZ21haWwuY29tIiwib3JpZ19pYXQiOjE1NjQ0OTI2ODl9.QgNpjP_2Vn0tXQ1_NLRTxmlJ5X9NqA2NFXIOWiomaiI" \
```
Comment (dmytrostriletskyi) [inline README.md]: Revert changes, please.

File: directory/user/endpoints.py [hunk 2]
```diff
@@ -23,4 +23,5 @@
     path('profile/', UserProfileSingle.as_view()),
     path('<str:username>/profile/', GetUserProfileSingle.as_view()),
     path('<str:username>/', UserSingle.as_view()),
+    path('<str:username>/', UserSingle.as_view()),
```
Comment (dmytrostriletskyi) [inline directory/user/endpoints.py]: Revert changes, please. Duplicated.

Comment (anastasiia-bilova) [reply directory/user/endpoints.py]: @dmytrostriletskyi, done.

File: README.md (hunk 1 above)
Comment (anastasiia-bilova) [reply README.md]: @dmytrostriletskyi, done.

**Pull request merged**